packages/services/api/src/modules/schema/providers/models/composite-legacy.ts

```diff
@@ -367,108 +358,4 @@ export class CompositeLegacyModel {
       reasons,
     };
   }
-
-  async delete({
```

Comment:
The `delete` feature was available for the legacy models for some reason, and it shouldn't, so I removed it.